### PR TITLE
Feature/add new case

### PR DESCRIPTION
**Pull request by bamfranceschi**
- Adds a component that lets users create new cases through the mobile app.
    - An "Add new case" button on the Family Connections screen routes users to a new screen.
    - This screen contains an input form for a social worker to fill out with details of a foster child (name, address, DOB, phone # etc).
    - At the bottom, there is a "save" button that lets users add the new case and the information is sent to the backend.
    - There is a "cancel" button that lets users back out of adding a new case.
    - The newly added case is displayed properly on the Family Connections screen.
    -  Feature includes tests using jest.
    - Includes error modal if something goes wrong in the process. This will later be updated with custom error messages.

### CODE REVIEW

File: src/screens/AddCaseScreen.test.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                .create(
+                    <Provider store={store}>
+                        <AddCaseScreen {...props} />
+                    </Provider>
+                )
+                .toJSON();
+            expect(tree?.children).toHaveLength(1);
+        });
+    });
+    test('matches snapshot', () => {
```
Comment (ccannell) [inline src/screens/AddCaseScreen.test.tsx:36]: We are not going to use snapshots for tests. They are brittle and more trouble than they are worth. Please remove.

**Comment by ccannell** [comment]
Your `package.json` file conflicts with master. You'll need to merge master into your branch and resolve the conflict.

**Comment by bamfranceschi** [comment]
Chris- turns out we do need to keep in place the undefined checks, because the Add Case form's photo is initially undefined.

**Comment by ccannell** [comment]
@bamfranceschi Two things:

1. There is a conflict. You'll need to merge master into your branch and resolve conflicts.
2. Remove snapshot test.

**Comment by ccannell** [comment]
Build is failing. Implicit any's

**Comment by bamfranceschi** [comment]
Yep, I'm working through them now

**Pull request closed without merging**